extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -15,6 +15,67 @@ const BackbeatConsumer = require('../../../lib/BackbeatConsumer');
 const BackbeatClient = require('../../../lib/clients/BackbeatClient');
 const QueueEntry = require('../utils/QueueEntry');
 
+class _UserAuthManager {
+    constructor(authConfig, log) {
+        assert.strictEqual(authConfig.type, 'user');
+
+        this.log = log;
+        const userInfo = authdata.users.find(
+            user => user.name === authConfig.user);
+        if (userInfo === undefined) {
+            throw Error(`No such user registered: ${authConfig.user}`);
```

Comment:
Maybe when we add the support for multiple users for the open source version, and a more clever lookup to match each role, we would want to be more gentle with misconfiguration and just ignore those entries not matching a registered user, to allow the user to miss some user info entries and still be able to replicate the other ones, for now I think it's ok since there can be only one configured user.